api/service/legacysync/downloader/build.sh

```diff
@@ -0,0 +1,4 @@
+
+# if command fails, try this
+# docker buildx create --use
+docker buildx build -t frozen621/harmony-proto:latest --platform linux/amd64,linux/arm64 -f Proto.Dockerfile --progress=plain .
```

Comment:
Can this be changed to `harmony-one` docker image?

Reply:
@sophoah , can we do this?

Reply:
@MaxMustermann2 i don't see why not if it is required, i'd just like to understand how where when is that build.sh is being used. Another PR explaining that would be great.

Reply:
@MaxMustermann2 i think what we wanted here. We definitely can.

Reply:
https://github.com/harmony-one/harmony/pull/4453